opm/output/eclipse/VectorItems/group.hpp

```diff
@@ -57,6 +57,7 @@ namespace Opm { namespace RestartIO { namespace Helpers { namespace VectorItems
     // the the IGRP vector, they should all be used as IGRP[NWGMAX + $index]
     enum index : std::vector<int>::size_type {
         ProdActiveCMode = 1,
+        ControlGroup = 5,
```

Comment:
In my opinion this should be `GroupControl` instead since the item describes a control mode as opposed to a group.

Reply:
Well - my understanding of the code was that this was the index of the controlling group - i.e. a group. But if I have misunderstood that I am more than happy to change.